galleon-pack/pom.xml

```diff
@@ -2248,8 +2270,8 @@
         </dependency>
 
         <dependency>
-            <groupId>org.hibernate</groupId>
-            <artifactId>hibernate-infinispan</artifactId>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-hibernate-cache-v53</artifactId>
```

Comment:
Is the other version no longer used? These two dependencies need to be 100% compatible.

Reply:
Hmm, I wonder if galleon-pack/pom should instead use the infinispan-hibernate-cache-v51 artifact that ORM 5.1 uses.  I'll try switching galleon-pack/pom + featurepack/pom.xml

Also, as mentioned above, Infinispan 9.2.x includes the equivalent of org.hibernate:hibernate-infinispan now, which we include in the org.infinispan.hibernate-cache module. If we are running EE 7, we use org.infinispan:infinispan-hibernate-cache-v51, if we are running EE 8, we use org.infinispan:infinispan-hibernate-cache-v53.  infinispan-hibernate-cache-v51 might be a better artifact to reference since that is for EE 7 mode.